### ISSUE TITLE
Using dataRetrieval versus waterData package

### ISSUE THREAD

**Issue by mkoohafkan** [USGS-R/dataRetrieval#110]
In the past I have used the `waterData` package (http://cran.r-project.org/web/packages/waterData/index.html) to access the USGS Daily Values Site Web Service. However, `waterData` is not listed among the USGS-R repositories. Does the `dataRetrieval` package supersede the `waterData` package? If not, are there any documents that provide guidance on the differences between these packages?

**Comment by ldecicco-USGS** [comment]
As far as getting USGS data from web services into R, `dataRetrieval` is more comprehensive (it includes all the web services, not just daily, and more flexible queries), and is the recommend package. Moving forward, this will be the package with the most on-going support. We are discussing modifying some of the `waterData` code to import `dataRetrieval`, but keep all backwards compatibility. 

`waterData` has other features that `dataRetrieval` does not, so depending on what you are using the package for, you may or may not have reason to alter any of your scripts (`fillMiss`, `plotAnoms`, `plotParams`, etc).

So, in a nutshell, if `waterData` suits your needs, there's no good reason to refactor your scripts. Moving forward, you may want to use `dataRetrieval` for the USGS data retrieval aspect for the added flexibility of the queries. 

Let me know if you need more information.

**Comment by ldecicco-USGS** [comment]
By the way, I was just playing with `kfigr` the other day! Cool package. I didn't get it up and running exactly, but it looks to be exactly what I need/want (I admittedly didn't spend too much time on it).

**Comment by mkoohafkan** [comment]
Thanks for the quick response! I'll try using `dataRetrieval` next time then. Also, I'd be happy to troubleshoot `kfigr` with you---feel free to post an issue on https://github.com/mkoohafkan/kfigr/issues, or email me directly. I'll bet there are still some kinks I need to work out.
